docker/php/config/parameters.yml

```diff
@@ -5,7 +5,7 @@ parameters:
     database_name: ${DATABASE_NAME:-symfony}
     database_user: ${DATABASE_USER:-root}
     database_password: ${DATABASE_PASSWORD:-~}
-    database_path: '${DATABASE_PATH:-"%kernel.project_dir%/data/db/wallabag.sqlite"}'
+    database_path: ${DATABASE_PATH:-"%kernel.project_dir%/data/db/wallabag.sqlite"}
```

Comment:
Before this change the generated value was `database_path: '"%kernel.project_dir%/data/db/wallabag.sqlite"'`, including the double quotes in the value